### DISCUSSION TITLE
ip180?

### DISCUSSION THREAD

**Discussion by mremedi2023** [ParadoxAlarmInterface/pai#469] [General]
Hello, any manual or instructions to connect ip180 with home assistant? Thank you

**Comment by zoltanhencsey** [comment]
Did you manage to connect with IP180?

**Comment by mremedi2023** [comment]
Hi, never, only ip150+

**Comment by zoltanhencsey** [comment]
Which firmware do you use in Ip150+ ?Üdvözlettel,Hencsey Zoltán
-------- Eredeti üzenet --------Feladó: mremedi2023 ***@***.***> Dátum: 2024. 10. 07.  15:46  (GMT+01:00) Címzett: ParadoxAlarmInterface/pai ***@***.***> Másolatot kap: zoltanhencsey ***@***.***>, Comment ***@***.***> Tárgy: Re: [ParadoxAlarmInterface/pai] ip180? (Discussion #469) 
Hi, never, only ip150+









El lun, 7 oct 2024 a la(s) 10:31 a.m., zoltanhencsey (

**Comment by mremedi2023** [comment]
The last 5.06.014


https://github.com/ParadoxAlarmInterface/pai/wiki/Compatibility
